src/helpers/errorHandlers.ts

```diff
@@ -137,3 +137,16 @@ export function checkEmptyResult(responses: Execution.IExecutionResponses) {
 
     return responses;
 }
+
+export const validateLocalIdentifiers = (buckets: VisualizationObject.IBucket[]) => {
```

Comment:
And I'd maybe rename it to hasDuplicateIdentifiers, as validate could mean more than finding duplicates

Reply:
done